.eslintrc.json

```diff
@@ -22,6 +22,11 @@
 			}
 		]
 	},
+	"settings": {
+		"jsdoc": {
+			"mode": "typescript"
+		}
+	},
```

Comment:
This addition sets ESLint's JSDoc parsing to analyze JSDoc comments using TypeScript. This is especially useful for our serverless function, where I use the `@type {import('...')}` syntax to bring in `@netlify/functions`' `Handler` type.